fixtures/vitest-pool-workers-examples/tsc-all.mjs

```diff
@@ -0,0 +1 @@
+// TODO(now): implement
```

Comment:
Vitest has some nice built in helpers and assertions for testing types, allowing you to scope errors and not have to test using @ts-expect-error in typescript files. Not sure what you had planned for this, but i've had some good success with the built ins from Vitest

Reply:
Gotcha, from vitest 1.0, you can run `vitest --typecheck` which does this under the hood. You can then use `*.test-d.ts` files specifically to write type tests with assertions. 

https://vitest.dev/guide/testing-types
https://vitest.dev/guide/testing-types#run-typechecking